manim/mobject/opengl_mobject.py

```diff
@@ -66,6 +66,8 @@ class OpenGLMobject:
     rgbas = _Data()
 
     is_fixed_in_frame = _Uniforms()
+    is_fixed_orientation = _Uniforms()
+    mob_center = _Uniforms()  # for fixed orientation reference
```

Comment:
I think for the sake of clarity it would make sense to rename `mob_center` to something more explicit like `fixed_mobject_center` or `fixed_center`.